nvtabular/ops.py

```diff
@@ -821,11 +806,9 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
 class FillMedian(DFOperator):
     """
     This operation replaces missing values with the median value for the column.
-
     Although you can directly call methods of this class to
     transform your continuous features, it's typically used within a
     Workflow class.
-
```

Comment:
Did you mean to remove these blank lines? We need them to render the docstrings correctly. With this change it looks like:

<img width="716" alt="Screen Shot 2020-07-02 at 9 38 37 AM" src="https://user-images.githubusercontent.com/69536/86387442-096c9680-bc48-11ea-9f03-272860559ab7.png">

when it should be something like:

<img width="744" alt="Screen Shot 2020-07-02 at 9 41 17 AM" src="https://user-images.githubusercontent.com/69536/86387561-3456ea80-bc48-11ea-9d12-5caee3d7d4bc.png">